appcenter-analytics/scripts/postlink.js

```diff
@@ -1,64 +1,19 @@
 const rnpmlink = require('appcenter-link-scripts');
-const inquirer = require('inquirer');
 
 // Configure Android first.
 let promise = null;
 if (rnpmlink.android.checkIfAndroidDirectoryExists()) {
     console.log('Configuring AppCenter Analytics for Android');
-    promise = rnpmlink.android.initAppCenterConfig()
-        .then(() =>
-            inquirer.prompt([{
-                type: 'list',
-                name: 'whenToEnableAnalytics',
-                message: 'For the Android app, should user tracking be enabled automatically?',
-                choices: [
-                    {
-                        name: 'Enable Automatically',
-                        value: 'ALWAYS_SEND'
-                    },
-                    {
-                        name: 'Enable in JavaScript',
-                        value: 'ENABLE_IN_JS'
-                    }]
-            }])
-        ).then((androidAnswer) => {
-            rnpmlink.android.patchStrings('appCenterAnalytics_whenToEnableAnalytics', androidAnswer.whenToEnableAnalytics);
-            rnpmlink.android.removeAndroidDuplicateLinks();
-        }).catch((e) => {
-            console.error(`Could not configure AppCenter Analytics for Android. Error Reason - ${e.message}`);
-            return Promise.resolve();
-        });
-} else {
-    promise = Promise.resolve();
+    rnpmlink.android.removeAndroidDuplicateLinks();
 }
+promise = Promise.resolve();
 
 // Then iOS even if Android failed.
 if (rnpmlink.ios.checkIfAppDelegateExists()) {
     promise
```

Comment:
I think we can replace it with just:
```suggestion
    rnpmlink.ios.initInAppDelegate('#import <AppCenterReactNativeAnalytics/AppCenterReactNativeAnalytics.h>', code, /.*\[AppCenterReactNativeAnalytics register.*/g).then // ...
```
And get rid of the `promise` variable (lines 9, 4, 34).

I'm not sure, but `return promise` on line 34 doesn't make sense. `module.exports = promise` should be used if we want to use this promise outside of this module.